server/passport-setup.js

```diff
@@ -5,21 +5,21 @@ dotenv.config({path: path.join(__dirname, './config/config.env') })
 const GoogleStrategy = require('passport-google-oauth20').Strategy;
 console.log('clientid', process.env);
 
-passport.use(new GoogleStrategy({
-    clientID: '366734958535-jj77akvpkh0vq6k0telp9elm42iv0cn5.apps.googleusercontent.com',
-    clientSecret: 'dXtJfgY64PXA1wQBb-eRXnED',
-    callbackURL: "http://localhost:3000/auth/google/callback"
-  },
-  function(accessToken, refreshToken, profile, cb) {
-    // User.findOrCreate({ googleId: profile.id }, function (err, user) {
-    //   return cb(err, user);
-    // });
-    console.log('access toke', accessToken);
-    console.log('refresh toke', refreshToken);
-    console.log('profile', profile);
-    return cb(null, profile);
-  }
-));
+// passport.use(new GoogleStrategy({
```

Comment:
Blocking: Would be good to remove this from the repo in your next commit to this branch since you have your secret hard coded into the passport setup which is a security issue. 

You may want to revoke this ID and secret and reissue them using the Google Cloud Platform and then update them in the config file. Happy to discuss tomorrow.